include/structures/cluster.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <vector>
+
+#include "structures/junction.hpp"  // for class Junction
+
+class Cluster
+{
+private:
+    std::vector<Junction> members{};
+
+public:
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    constexpr Cluster()                    = default; //!< Defaulted.
+    Cluster(Cluster const &)               = default; //!< Defaulted.
+    Cluster(Cluster &&)                    = default; //!< Defaulted.
+    Cluster & operator=(Cluster const &)   = default; //!< Defaulted.
+    Cluster & operator=(Cluster &&)        = default; //!< Defaulted.
+    ~Cluster()                             = default; //!< Defaulted.
+    //!\}
+
+    Cluster(std::vector<Junction> members) : members{std::move(members)}
+    {
+
+    }
+    size_t get_cluster_size() const;
+
+    Breakend get_average_mate1() const;
+
+    Breakend get_average_mate2() const;
```

Comment:
and here

Reply:
👍 -> https://github.com/seqan/iGenVar/issues/70